### PR TITLE
Update proposal process with steps and board

### PR DESCRIPTION
**Pull request by OrlinVasilev**
Signed-off-by: OrlinVasilev <ovasilev@vmware.com>

Thank you for contributing to Harbor!

# Comprehensive Summary of your change

# Issue being fixed
Fixes #(issue)

Please indicate you've done the following:
- [x] Well Written Title and Summary of the PR
- [x] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [ ] Made sure tests are passing and test coverage is added if needed.
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Comment by OrlinVasilev** [comment]
@a-mccarthy can you check the wording please :)

**Comment by Vad1mo** [comment]
Develop and propose new features.

- The following simple step process can be used to submit new features or changes to the existing code.
- See if your feature is already being worked on. Check both the [Issues](https://github.com/goharbor/harbor/issues) and the [PRs](https://github.com/goharbor/harbor/pulls) in the main Harbor repository as well as the [Community repository](https://github.com/goharbor/community).
- Submit (open PR) with the new proposal in [community/proposals/new](https://github.com/goharbor/community/tree/main/proposals/new) using this [template](https://github.com/goharbor/community/blob/main/proposals/TEMPLATE.md).
- The proposal must be labeled as "kind/proposal" once submitted. Check examples [here](https://github.com/goharbor/community/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+label%3Akind%2Fproposal).
- The proposal can be modified and adapted to meet the requirements from the community and other maintainers and contributors. The overall architecture needs to be consistent to avoid duplicate work in the Roadmap.
- When reviewed and approved, it can be implemented either by the original submitter or by anyone else in the community. Open PRs in the respective repositories with all the necessary code and test changes!
- Once implemented or during the implementation, the PRs are reviewed by maintainers and contributors, following the currently enabled practices and methods.
- After merging the new PRs, the proposal must be moved to Community/Proposals and marked as done.
- You have made Harbor even better, congratulations. Thank you!


> sorry a few links got lost.

**Comment by OrlinVasilev** [comment]
@Vad1mo adopted your input ! Thank you!

**Comment by OrlinVasilev** [comment]
@KaulAnmol check this out :)

**Review by Vad1mo — APPROVED** [review]

**Comment by OrlinVasilev** [comment]
Hey Folks can you check that please @wy65701436 @KaulAnmol @holyhope thanks!

File: CONTRIBUTING.md
Comment (thcdrt) [inline CONTRIBUTING.md:382]: Are links being part of `### The following simple process can be used to submit new features or changes to the existing code. ?`
If no we should put them under a dedicated title like `Useful links` ?

File: CONTRIBUTING.md
Comment (thcdrt) [inline CONTRIBUTING.md:373]:
```suggestion
- The proposal can be modified and adapted to meet the requirements from the community, other maintainers and contributors. The overall architecture needs to be consistent to avoid duplicate work in the [Roadmap](https://github.com/goharbor/harbor/wiki#roadmap).
```

File: CONTRIBUTING.md
Comment (thcdrt) [inline CONTRIBUTING.md:375]:
```suggestion
- When reviewed and approved it can be implemented either by the original submitter or anyone else from the community which we highly encourage, as the project is community driven. Open PRs in the respective repositories with all the necessary code and test changes as described in the current document.
```

File: CONTRIBUTING.md
Comment (thcdrt) [inline CONTRIBUTING.md:376]: The word `enabled` is weird here ? Maybe replace it by `best` ?

**Review by thcdrt — APPROVED** [review]

**Review by wy65701436 — APPROVED** [review]
lgtm

**Pull request merged**
